README.md

```diff
@@ -7,182 +7,119 @@
 [![Coverage Status](https://coveralls.io/repos/github/linthtml/linthtml/badge.svg?branch=develop)](https://coveralls.io/github/linthtml/linthtml?branch=develop)
 <!-- [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Flinthtml%2Flinthtml.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Flinthtml%2Flinthtml?ref=badge_shield) -->
 
-> An unofficial html5 linter and validator.
+> An unofficial HTML5 linter and validator.
 
-LintHTML is a fork of [htmllint](https://github.com/htmllint/htmllint) with an integrated CLI, all htmllint's rules can be used with LintHTML.
-The migration from htmllint to LintHTML is easy, just follow this [guide](htmllint-migration).
-<!-- _LintHTML uses [htmlparser2](https://www.npmjs.com/package/htmlparser2) to parse your html._ -->
+LintHTML is a fork of [htmllint](https://github.com/htmllint/htmllint) featuring a built-in CLI and multiple bug fixes and improvements.
+The migration from htmllint to LintHTML is easy, as all htmllint's rules can be used with LintHTML – just follow this [simple guide](#migrate-from-htmllint).
 
 ## Installation and Usage
 
-Prerequisites:
-
-* [Node.js](https://nodejs.org/en/) 6+
-* [npm](https://npmjs.com) version 3+
-
-There are two ways to install LintHTML: globally and locally.
-
-### Local Installation and Usage
-
-If you want to include LintHTML as part of your project's build system, we recommend installing it locally. You can do so using npm or yarn:
+You can install LintHTML either locally or globally.
+For most cases we recommend the former, which can be achieved this way with npm:
```

Comment:
Nice but we should include a link to a doc file about `installation and usage`

Reply:
You mean a independent file, don't you? Sounds OK.

I deleted the global installation part because it makes little sense these days that we're able to run things with npx, and I'm not sure if it's a good practice to encourage people to install it globally. Anyway, if you're thinking in an extended `installation and usage` doc, it can fit there :).

Reply:
A independent file yes.
Most people will use `npx` yes but not everyone, that's why I think it could be a good idea to have a separate file with differents installations and complex usage examples.

Reply:
Done! Please take a look and tell me if you'd add anything else :).